scaffold/convert/create-test.template.js

```diff
@@ -0,0 +1,30 @@
+require('should');
+
+const zapier = require('zapier-platform-core');
+
+const App = require('../../index');
+const appTester = zapier.createAppTester(App);
+
+const utils = require('../utils');
+
+describe('Creates - <%= KEY %>', () => {
+  zapier.tools.env.inject();
+
+  it('should create an object', (done) => {
+    const bundle = {
+      authData: utils.getAuthData(),
+      inputData: {
+        name: 'Testing',
+      },
+    };
+
+    appTester(App.creates.<%= KEY %>.operation.perform, bundle)
```

Comment:
Is there a possibility of `KEY` having a `-`? If so, we may prefer doing `['<%= KEY %>']` here.

Reply:
Good catch. Fixed in 0f46d2b.